.github/CODEOWNERS

```diff
@@ -387,7 +387,7 @@
 # New structures team.
 # Question project.
 /core/controllers/practice_sessions*.py @aks681
-/core/controllers/question*.py @aks681
+/core/controllers/question*.py @aks681 @rohitkatlaa
```

Comment:
Ditto here*

Reply:
I wanted to gain experience with code review and since I am currently a mentor of the project that deals with controllers so I thought I could become a codeowner for a few controller files.